css-two-column-layout/styles.css

```diff
@@ -0,0 +1,57 @@
+body {
+  font-family: Karla, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: 40px auto;
+}
+
+.row {
+  display: flex;
+  margin: 15px 0;
+}
+
+.column-full {
+  max-width: 70%;
```

Comment:
a column full should have a set width of 100%. If you need to control the children, you can use display flex on this element.